py/desiutil/names.py

```diff
@@ -42,15 +42,16 @@ def radec_to_desiname(target_ra, target_dec):
     # Convert to numpy array in case inputs are scalars or lists
     target_ra, target_dec = np.atleast_1d(target_ra), np.atleast_1d(target_dec)
 
-    inputs = {'target_ra': target_ra, 'target_dec': target_dec}
-    tests = (('NaN values', np.isnan),
-             ('Infinite values', np.isinf),
-             ('RA not in range [0, 360)', lambda x: (x < 0) | (x >= 360)),
-             ('Dec not in range [-90, 90]', lambda x: (x < -90) | (x > 90)))
+    base_tests = [('NaN values', np.isnan),
+                  ('Infinite values', np.isinf),]
+    inputs = {'target_ra': {'data': target_ra,
+                            'tests': base_tests + [('RA not in range [0, 360)', lambda x: (x < 0) | (x >= 360))]},
+              'target_dec': {'data': target_dec,
+                             'tests': base_tests + [('Dec not in range [-90, 90]', lambda x: (x < -90) | (x > 90))]}}
     for i in inputs:
```

Comment:
Could we change `i` to `coord` or `ra_or_dec` or something a little more informative? It took me a bit to parse the loops and tuple unpacking, so anything helps for readability.

Reply:
That's fine. I'll change it to `coord`.